src/sentry/api/endpoints/organization_events_trendsv2.py

```diff
@@ -168,7 +175,7 @@ def get_event_stats_metrics(_, user_query, params, rollup, zerofill_results, __)
                 request,
                 organization,
                 get_event_stats_metrics,
-                top_events=top_events_limit,
+                top_events=events_per_query,
```

Comment:
passing `events_per_query` so that returned data can be processed as if this was a top n request